### ISSUE TITLE
Default showcases of Material and Cupertino Theme

### ISSUE THREAD

**Issue by LucasJosefiak** [widgetbook/widgetbook#688]
**Description**
Some kind of page or set of pages that without further configuration showcases your Material or Cupertino theme would be great.
Like display the Fonts, the Colors, and the standard Flutter input widgets with your theme.
Pretty much all apps are using Material (with custom components on top), so this would be immediately useful for anyone 

**Additional Context**
This is a copied message from our Discord user `bent`

**Comment by ABausG** [comment]
https://medium.com/widgetbook/getting-material-base-components-into-widgetbook-c0e1f874b14b Guess this could be a starting point already? ;)

**Comment by LucasJosefiak** [comment]
True! Great that you found this issue😄 @YoussefRaafatNasry what do you think?

**Comment by YoussefRaafatNasry** [comment]
@ABausG This will be definitely a great start for material themes, but we are looking into an approach to support custom themes first, then use the same approach to support material and cupertino themes.
